dom.bs

```diff
@@ -1526,9 +1527,7 @@ The <dfn export id=concept-node-length for=Node>length</dfn> of a
  <dt>{{DocumentType}}
  <dd>Zero.
 
- <dt>{{Text}}
- <dt>{{ProcessingInstruction}}
- <dt>{{Comment}}
+ <dt>{{CharacterData}}
```

Comment:
See https://github.com/whatwg/dom/commit/443023ad7e5a04d050e21a206aff1b5bd66ddc07 for why I decided not to do this.